app/components/button-with-confirmation/component.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  isConfirmed: null,
+  isUnconfirmed: Ember.computed.not('isConfirmed'),
+  input: function(){
+    var enteredValue = this.get('enteredValue');
+    var confirmValue = this.get('confirmValue');
+    this.set('isConfirmed', enteredValue === confirmValue);
+  }.on('init'),
```

Comment:
Might be clearer to make `isConfirmed` a cp?

```
isConfirmed: function(){
  return this.get('enteredValue') === this.get('confirmValue');
}.property('enteredValue', 'confirmValue')
```

I had actually forgotten that `input` was an event handler, so I didn't realize what was happening here.